src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -599,15 +599,16 @@ private void drawTurnCounter(Graphics g, boolean counterIsRight)
     if( lastTurnNum != turnNum )
     {
       lastTurnNum = turnNum;
+      PixelFont pf = SpriteLibrary.getFontStandard();
       BufferedImage day = SpriteUIUtils.getTextAsImage("Turn ");
       BufferedImage dayNum = SpriteUIUtils.getBoldTextAsImage(Integer.toString(turnNum));
       int width = day.getWidth() + dayNum.getWidth();
-      int height = dayNum.getHeight();
+      int height = Math.max(pf.getAscent(), dayNum.getHeight());
 
       turnNumImage = SpriteLibrary.createTransparentSprite(width, height);
       Graphics dcg = turnNumImage.getGraphics();
       dcg.drawImage(day, 0, 0, null);
-      dcg.drawImage(dayNum, turnNumImage.getWidth()-dayNum.getWidth(), 0, null);
+      dcg.drawImage(dayNum, turnNumImage.getWidth()-dayNum.getWidth(), pf.getAscent()-dayNum.getHeight(), null);
```

Comment:
I'm not a huge fan of how this looks, but I feel like moving the number away from bottom-aligned makes it look even weirder.
![Turn 2 fancy](https://cdn.discordapp.com/attachments/358161783208411136/793790441359278120/unknown.png)
Might consider just using normal numbers instead of the fancy number?
![Turn 2 bland](https://cdn.discordapp.com/attachments/358161783208411136/793791146685104148/unknown.png)

Reply:
I like the bold numbers because they stand out more, but I'm not too tied to it either way.